### PR TITLE
Unit Tests for twophase

### PR DESCRIPTION
**Pull request by oraluben**
UCP #266 
### What problem does this PR solve? 
#266

### Check List 

Tests 

 - Unit test

Code changes

 - Has Go code change

Side effects

Related changes

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: controllers/test/types.go [hunk 1]
```diff
@@ -4,13 +4,11 @@ import (
...
-	"k8s.io/apimachinery/pkg/runtime"
-
-	v1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
 	"github.com/golang/protobuf/ptypes/empty"
 	"google.golang.org/grpc"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
```
Comment (fewdan) [inline controllers/test/types.go]: please group go imports

Comment (oraluben) [reply controllers/test/types.go]: What principle or standard should I follow to group go imports? I tried `make groupimports` and it says it's ok.

Comment (YangKeao) [reply controllers/test/types.go]: It looks well. Where is the problem? @fewdan

**Comment by oraluben** [comment]
@lucklove @cwen0 PTAL

**Pull request merged**